### PR TITLE
doc update

### PR DESCRIPTION
**Pull request by ring630**

### CODE REVIEW

File: doc/source/Getting_started/Installation.rst [hunk 1]
```diff
@@ -135,4 +135,34 @@ To use IronPython in AEDT:
...
+Install required packages.
+
+Download `requirements.txt <https://github.com/pyansys/pyaedt/blob/main/requirements/requirements.txt>`_
+
+.. code::
+
+    (pyaedt_py310) C:\>pip install -r requirements.txt
+
+Upgrade pyaedt to the latest version
+~~~~~~~~~~~~~~~~~~~~~~
```
Comment (PipKat) [inline doc/source/Getting_started/Installation.rst:164]:
```suggestion
Upgrade PyAEDT to the latest version
~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/source/Getting_started/Installation.rst [hunk 2]
```diff
@@ -135,4 +135,34 @@ To use IronPython in AEDT:
 
 .. code::
 
-    ipy64 setup-distutils.py install --user
\ No newline at end of file
+    ipy64 setup-distutils.py install --user
+
+
+Install pyaedt in Conda virtual environment
```
Comment (PipKat) [inline doc/source/Getting_started/Installation.rst:141]:
```suggestion
Install PyAEDT in Conda virtual environment
```

File: doc/source/Getting_started/Installation.rst [hunk 3]
```diff
@@ -135,4 +135,34 @@ To use IronPython in AEDT:
...
-    ipy64 setup-distutils.py install --user
\ No newline at end of file
+    ipy64 setup-distutils.py install --user
+
+
+Install PyAEDT in Conda virtual environment
+~~~~~~~~~~~~~~~~~~~~~~
+Create virtual environment
+
+.. code::
```
Comment (MaxJPRey) [inline doc/source/Getting_started/Installation.rst:145]: .. code:: bash

File: doc/source/Getting_started/Installation.rst [hunk 4]
```diff
@@ -135,4 +135,34 @@ To use IronPython in AEDT:
...
+
+    (base) C:\>conda create --name pyaedt_py310 python=3.10
+
+Activate virtual environment
+
+.. code::
```
Comment (MaxJPRey) [inline doc/source/Getting_started/Installation.rst:151]:
```suggestion
.. code:: bash
```

File: doc/source/Getting_started/Installation.rst (hunk 3 above)
Comment (MaxJPRey) [inline doc/source/Getting_started/Installation.rst:145]:
```suggestion
.. code:: bash
```

File: doc/source/Getting_started/Installation.rst [hunk 5]
```diff
@@ -135,4 +135,34 @@ To use IronPython in AEDT:
...
+
+    (base) C:\>conda activate pyaedt_py310
+
+Install required packages.
+
+Download `requirements.txt <https://github.com/pyansys/pyaedt/blob/main/requirements/requirements.txt>`_
+
+.. code::
```
Comment (MaxJPRey) [inline doc/source/Getting_started/Installation.rst:159]:
```suggestion
.. code:: bash
```

File: doc/source/Getting_started/Installation.rst [hunk 6]
```diff
@@ -135,4 +135,34 @@ To use IronPython in AEDT:
...
+
+    (pyaedt_py310) C:\>pip install -r requirements.txt
+
+Upgrade PyAEDT to the latest version
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code::
```
Comment (MaxJPRey) [inline doc/source/Getting_started/Installation.rst:166]:
```suggestion
.. code:: bash
```

File: doc/source/Getting_started/Installation.rst [hunk 7]
```diff
@@ -135,4 +135,34 @@ To use IronPython in AEDT:
...
+
+    (base) C:\>conda activate pyaedt_py310
+
+Install required packages.
+
+Download `requirements.txt <https://github.com/pyansys/pyaedt/blob/main/requirements/requirements.txt>`_
+
+.. code::
+
+    (pyaedt_py310) C:\>pip install -r requirements.txt
```
Comment (MaxJPRey) [inline doc/source/Getting_started/Installation.rst:161]: To allow easy copy/paste of the code sample.
```suggestion
    pip install -r requirements.txt
```

File: doc/source/Getting_started/Installation.rst [hunk 8]
```diff
@@ -135,4 +135,34 @@ To use IronPython in AEDT:
...
+
+.. code::
+
+    (base) C:\>conda create --name pyaedt_py310 python=3.10
+
+Activate virtual environment
+
+.. code::
+
+    (base) C:\>conda activate pyaedt_py310
```
Comment (MaxJPRey) [inline doc/source/Getting_started/Installation.rst:153]: To allow easy copy/paste of the code sample.
```suggestion
    conda activate pyaedt_py310
```

File: doc/source/Getting_started/Installation.rst [hunk 9]
```diff
@@ -135,4 +135,34 @@ To use IronPython in AEDT:
...
+    ipy64 setup-distutils.py install --user
+
+
+Install PyAEDT in Conda virtual environment
+~~~~~~~~~~~~~~~~~~~~~~
+Create virtual environment
+
+.. code::
+
+    (base) C:\>conda create --name pyaedt_py310 python=3.10
```
Comment (MaxJPRey) [inline doc/source/Getting_started/Installation.rst:147]: To allow easy copy/paste of the code sample.
```suggestion
    conda create --name pyaedt_py310 python=3.10
```

File: doc/source/Getting_started/Installation.rst [hunk 10]
```diff
@@ -135,4 +135,34 @@ To use IronPython in AEDT:
...
+.. code::
+
+    (pyaedt_py310) C:\>pip install -r requirements.txt
+
+Upgrade PyAEDT to the latest version
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code::
+
+    (pyaedt_py310) pip install -U pyaedt
```
Comment (MaxJPRey) [inline doc/source/Getting_started/Installation.rst:168]: To allow easy copy/paste of the code sample.
```suggestion
    pip install -U pyaedt
```

**Comment by ring630** [comment]
@MaxJPRey could you help me on this error? What does this mean?
`/github/home/vale --output=/lib/rdjsonl.tmpl --config=doc/.vale.ini doc
  {"message": "[Vale.Spelling] Did you really mean 'Conda'?", "location": {"path": "doc/source/Getting_started/Installation.rst", "range": {"start": {"line": 141, "column": 19}}}, "severity": "ERROR"}
  /bin/reviewdog -f=rdjsonl -name=vale -reporter=github-pr-check -fail-on-error=true -filter-mode=nofilter -level=error
  2023/02/25 05:36:32 [vale] reported: https://github.com/pyansys/pyaedt/runs/1159315[30](https://github.com/pyansys/pyaedt/actions/runs/4268565939/jobs/7431141173#step:4:31)18 (conclusion=failure)`

**Comment by MaxJPRey** [comment]
> @MaxJPRey could you help me on this error? What does this mean? `/github/home/vale --output=/lib/rdjsonl.tmpl --config=doc/.vale.ini doc {"message": "[Vale.Spelling] Did you really mean 'Conda'?", "location": {"path": "doc/source/Getting_started/Installation.rst", "range": {"start": {"line": 141, "column": 19}}}, "severity": "ERROR"} /bin/reviewdog -f=rdjsonl -name=vale -reporter=github-pr-check -fail-on-error=true -filter-mode=nofilter -level=error 2023/02/25 05:36:32 [vale] reported: https://github.com/pyansys/pyaedt/runs/1159315[30](https://github.com/pyansys/pyaedt/actions/runs/4268565939/jobs/7431141173#step:4:31)18 (conclusion=failure)`

It is because `Conda` is a proper noun and thus not a common noun allowed by Vale -the tool used to check the typos, grammar... in our documentation-.
I just added `Conda` to the `accept.txt` file to notify Vale that it is intended and not a typo.

**Review by maxcapodi78 — APPROVED** [review]

**Pull request merged**